docs/data/base/components/select/select.md

```diff
@@ -132,6 +132,15 @@ The following code snippet applies a CSS class called `my-listbox` to the listbo
 <Select slotProps={{ listbox: { className: 'my-listbox' } }} />
 ```
 
+### Portals
+
+By default, the Select's popup is rendered in a portal and appended to the bottom of the DOM.
```

Comment:
```suggestion
By default, the Select's popup is rendered in a [portal](https://mui.com/base-ui/react-portal/components-api/) and appended to the bottom of the DOM.
```

i think it might be good to add link to portal here, it will be helpful for people who are starting with react as they might not know what portal means.